<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ITEMINFO</class>
 <widget class="QWidget" name="ITEMINFO">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1431</width>
    <height>897</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ITEMINFO</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="le_impath"/>
         </item>
         <item>
          <widget class="QPushButton" name="pb_read">
           <property name="text">
            <string>读取</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le_expath"/>
         </item>
         <item>
          <widget class="QPushButton" name="pb_write">
           <property name="text">
            <string>导出</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="le_name">
           <property name="text">
            <string>string</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le_id">
           <property name="text">
            <string>输入家具id</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_wline">
           <property name="text">
            <string>增加</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_dline">
           <property name="text">
            <string>删除</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="tableView"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
